File: app/src/main/java/mb00/android/codehub/ui/home/inject/HomePulseFragmentComponent.java
package mb00.android.codehub.ui.home.inject;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;
import mb00.android.codehub.ui.user.view.HomePulseFragment;


@Subcomponent(modules = HomePulseFragmentModule.class)
public interface HomePulseFragmentComponent extends AndroidInjector<HomePulseFragment> {

    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<HomePulseFragment> {}

}

File: app/src/main/java/mb00/android/codehub/ui/home/inject/HomePulseFragmentComponent.kt
package mb00.android.codehub.ui.home.inject

import dagger.Subcomponent
import dagger.android.AndroidInjector
import mb00.android.codehub.ui.user.view.HomePulseFragment


@Subcomponent(modules = [
    HomePulseFragmentModule::class
])
interface HomePulseFragmentComponent : AndroidInjector<HomePulseFragment> {

    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<HomePulseFragment>()

}
